src/components/Modals/Send/Form.tsx

```diff
@@ -90,6 +84,10 @@ export const Form: React.FC<SendFormProps> = ({ initialAssetId, input = '', acco
       txHash: '',
     },
   })
+  const assetId = useWatch<SendInput, SendFormFields.AssetId>({
+    name: SendFormFields.AssetId,
+    control: methods.control,
+  })
```

Comment:
Uses this in addition to initialAssetId as a directive on whether to render the select route, instead of a redirect causing issues